### PR TITLE
[ricos-styles] fix: color decoration

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: packages/ricos-editor/web/src/tiptap/theme-defaults.ts [hunk 1]
```diff
@@ -1,74 +1,77 @@
 // reference for values is in consts.scss in common package
-import type { RicosCustomStyles } from 'ricos-types';
+import type { RicosTheme } from 'ricos-types';
 
-const defaultThemeValues = { fontWeight: 'normal', fontStyle: 'normal', lineHeight: '1.5' };
+const defaultCustomStyle = { fontWeight: 'normal', fontStyle: 'normal', lineHeight: '1.5' };
+const defaultPalette = { bgColor: '#ffffff', textColor: '#000000' };
```
Comment (lxgreen) [inline packages/ricos-editor/web/src/tiptap/theme-defaults.ts:5]: nice!

File: packages/ricos-styles/web/src/decorations/color-decoration.ts [hunk 2]
```diff
@@ -19,7 +20,8 @@ export class ColorDecoration implements TextDecoration {
     }
 
     const { foreground: color, background: backgroundColor } = decoration.colorData || {};
-    return new ColorDecoration({ color, backgroundColor });
+    const customStyle = pickBy({ color, backgroundColor });
```
[the remark this answers is not in the document]
Comment (lxgreen) [reply packages/ricos-styles/web/src/decorations/color-decoration.ts:23]: and if you want remove `undefined`:
```
const customStyle = [color, background].filter(Boolean);
```

[the remark this answers is not in the document]
Comment (lxgreen) [reply packages/ricos-styles/web/src/decorations/color-decoration.ts:23]: lodash is bad

**Review by lxgreen — CHANGES_REQUESTED** [review]
some style questions

File: packages/ricos-styles/web/src/decorations/color-decoration.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (maimonav) [reply packages/ricos-styles/web/src/decorations/color-decoration.ts:23]: If it was the same result, I wouldn't add this change :)
The problem is that undefined is being added to the object, and when afterwards, the undefined overrides actual value.
Do you have a better style to remove undefined? I WOULD LOVE TO KNOW ABOUT IT

[the remark this answers is not in the document]
Comment (maimonav) [reply packages/ricos-styles/web/src/decorations/color-decoration.ts:23]: And you and me both, I'm not a fan of lodash

**Pull request merged**